ui/src/containers/Header/Header.test.tsx

```diff
@@ -12,6 +12,14 @@ const TESTDATA_DIR = `src/store/testdata`;
 const api = new FakeHub(TESTDATA_DIR);
 const { Provider, root } = createProviderAndStore(api);
 
+beforeEach(() => {
```

Comment:
Can we please add some description in the comment to understand what `beforeEach`  and `afterEach` actually does and why do we need them ?